SU2_CFD/src/hybrid_RANS_LES_model.cpp

```diff
@@ -351,21 +352,22 @@ void CHybrid_Mediator::SetupResolvedFlowSolver(const CGeometry* geometry,
                                                mean_eddy_visc,
                                                aniso_eddy_viscosity);
 
-    /*--- XXX: This is an ad-hoc correction
-     * Rescale the eddy viscosity to rapidly remove fluctuations where
-     * resolved turbulence has been transported into regions with
-     * insufficient resolution ---*/
-
-    const su2double avg_resolution_adequacy =
-        solver_container[FLOW_SOL]->average_node[iPoint]->GetResolutionAdequacy();
-    assert(avg_resolution_adequacy >= 0);
-    assert(avg_resolution_adequacy == avg_resolution_adequacy);
-    const su2double factor = pow(min(avg_resolution_adequacy, 10.0), 4.0/3);
-    for (unsigned short iDim = 0; iDim < nDim; iDim++) {
-      for (unsigned short jDim = 0; jDim < nDim; jDim++) {
-         aniso_eddy_viscosity[iDim][jDim] *= factor;
-      }
-    }
+    // /*--- XXX: This is an ad-hoc correction
+    //  * Rescale the eddy viscosity to rapidly remove fluctuations where
+    //  * resolved turbulence has been transported into regions with
+    //  * insufficient resolution ---*/
+
+    // const su2double avg_resolution_adequacy =
+    //   solver_container[FLOW_SOL]->average_node[iPoint]->GetResolutionAdequacy();
+    // assert(avg_resolution_adequacy >= 0);
+    // assert(avg_resolution_adequacy == avg_resolution_adequacy);
+    // //const su2double factor = pow(min(avg_resolution_adequacy, 10.0), 4.0/3);
+    // const su2double factor = max(min(avg_resolution_adequacy, 10.0), 1.0);
+    // for (unsigned short iDim = 0; iDim < nDim; iDim++) {
+    //   for (unsigned short jDim = 0; jDim < nDim; jDim++) {
+    //      aniso_eddy_viscosity[iDim][jDim] *= factor;
+    //   }
+    // }
```

Comment:
@trevilo Is there a reason this was removed?  Was it also removed in CDP?

Reply:
No, it is still in CDP.  I commented it out for the channel runs for two reasons: 1) it is hacky so I just wanted to see if we really needed it (at least for the channel), and 2) for stability again.  Now, as before, the stability issues were observed prior to any implicitness on the SGET term.  So, I was trying to keep mu^{SGET} from getting big.  That shouldn't really be a concern anymore, but I have not tested with this back on.